commnuity-calls/template/README.md

```diff
@@ -0,0 +1,9 @@
+# Template
+
+The purpose of the template community call is to provide a template for running your own community call.
+
+## Meetings
+
+| No. |Date | Host | Topic | Agenda | Notes | Recording |
+|:---|:---|:---|---|---|---|:---|
+1 | Today | Liebniz | Example Topic | [agenda](https://example.com) | [write up](https://example.com) | [video](https://example.com) |
```

Comment:
Happy to take lead here.

[Calendar link](https://calendar.google.com/calendar/render?cid=c_7a8435203986a3f022dcfb47c17f3e238e3aa0659e1523b22ee668193d2ad880@group.calendar.google.com)

We can do the first one after EthCC.

I propose the topic is "MEV SoK". I basically want to look back at different ways of describing MEV from Phil and Xyn, add some more of my own spice, look at things like Peter's tweet, and open up a discussion that will help community members understand the breadth and depth of the design space SUAVE is trying to navigate before we dive into SUAVE-specific stuff in future calls.